models/issues/milestone_label.go

```diff
@@ -0,0 +1,305 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package issues
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"code.gitea.io/gitea/models/db"
+	access_model "code.gitea.io/gitea/models/perm/access"
+	user_model "code.gitea.io/gitea/models/user"
+)
+
+// MilestoneLabel represents an milestone-label relation.
+type MilestoneLabel struct {
+	ID          int64 `xorm:"pk autoincr"`
+	MilestoneID int64 `xorm:"UNIQUE(milestoneid_labelid)"`
+	LabelID     int64 `xorm:"UNIQUE(milestoneid_labelid)"`
+}
+
+func init() {
+	db.RegisterModel(new(MilestoneLabel))
```

Comment:
on repo deletion, repo lables get deleted, we need to make sure that milestoneLabel that reference to them get deleted too

Reply:
Added deletion of MilestoneLabels